src/lib.rs

```diff
@@ -557,6 +551,19 @@ mod tests {
 
     }
 
+    #[test]
+    fn parse_trailing_semi() {
```

Comment:
I know that you've probably used the other tests to come up with this name, but I think it's a bit unclear.

I feel like `csi_parse_trailing_semicolon` might be a bit clearer here.

Reply:
Sure, I was just trying to pick up on the convention :)

Reply:
> Sure, I was just trying to pick up on the convention :)

Yeah, which I think was a good idea. But I feel like the conventions used here aren't optimal here.